File: PartnerApi/C#/b2_eject_group_member.cs
using System.Text;

class Program
{
    static readonly HttpClient Http = new();

    static async Task Main()
    {
        const string apiUrl = ""; // Provided by b2_authorize_account
        const string authToken = ""; // Provided by b2_authorize_account
        const string adminAccountId = ""; // Provided by b2_authorize_account
        const string groupId = ""; // Provided by b2_list_groups
        const string memberAccountId = ""; // Provided by b2_create_group_member or b2_list_group_members

        var postData = $@" {{ ""adminAccountId"": ""{adminAccountId}"", ""groupId"": ""{groupId}"", ""memberAccountId"": ""{memberAccountId}"" }} ";

        Http.DefaultRequestHeaders.TryAddWithoutValidation("Authorization", authToken);

        var response = await Http.PostAsync(
            $"{apiUrl}/b2api/v4/b2_eject_group_member",
            new StringContent(postData, Encoding.UTF8, "application/json")
        );

        Console.WriteLine(await response.Content.ReadAsStringAsync());
    }
}

File: PartnerApi/PHP/b2_eject_group_member.php
<?php

$api_url = ''; // Provided by b2_authorize_account
$auth_token = ''; // Provided by b2_authorize_account
$admin_account_id = ''; // Provided by b2_authorize_account
$group_id = ''; // Provided by b2_list_groups
$member_account_id = ''; // Provided by b2_create_group_member or b2_list_group_members

$postData = json_encode([
    'adminAccountId' => $admin_account_id,
    'groupId' => $group_id,
    'memberAccountId' => $member_account_id,
]);

$ch = curl_init($api_url . '/b2api/v4/b2_eject_group_member');
curl_setopt_array($ch, [
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_POST           => true,
    CURLOPT_HTTPHEADER     => [
        "Authorization: $auth_token",
        'Content-Type: application/json',
    ],
    CURLOPT_POSTFIELDS     => $postData,
]);

echo curl_exec($ch), PHP_EOL;
